Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -51,6 +51,10 @@ public struct Platform: Encodable, Equatable {
     /// The WebAssembly System Interface platform.
     @available(_PackageDescription, introduced: 5.3)
     public static let wasi: Platform = Platform(name: "wasi")
+
+    /// The OpenBSD platform.
+    @available(_PackageDescription, introduced: 5.4)
```

Comment:
@abertelrud what should the "introduced" be in this case? 999?

Reply:
Yah, should be 999 for anything new and we will update it to whatever the next version will be once we know.

Reply:
Done.